lib/functoria/install.ml

```diff
@@ -40,11 +40,17 @@ let pp ppf t =
   Fmt.pf ppf "etc: [%s%s]" (String.concat "" etcs)
     (match etcs with [] -> "" | _ -> "\n")
 
-let pp_opam ppf t =
+let pp_opam ?subdir () ppf t =
   let pp_bin ppf (src, dst) =
-    Fmt.pf ppf {|"cp" "dist/%a" "%%{bin}%%/%a"|} Fpath.pp src Fpath.pp dst
+    Fmt.pf ppf {|"cp" "%adist/%a" "%%{bin}%%/%a"|}
+      Fmt.(option ~none:(any "") (Fpath.pp ++ any "/"))
```

Comment:
`Fpath.pp ++ any "/"` can output a `//` when `subdir` already ends with a `/`. 

It's the case in the example opam file that you linked: https://git.robur.io/hannes/test-opam/src/branch/main/packages/mirage-unikernel-hello-hvt.0.0.1/opam#L18

Reply:
done in e62652ba